src/mjml/mjml.service.js

```diff
@@ -60,4 +65,18 @@ export default class MjmlService {
       return null;
```

Comment:
please always return the same type. `return '';`

Reply:
done